### PR TITLE
Remove selected fields from the observation plan

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR removes the selected fields from the observation plan.

Co-authored with @jadalilleboe.

### CODE REVIEW

File: skyportal/handlers/api/observation_plan.py [hunk 1]
```diff
@@ -904,6 +904,81 @@ def get(self, observation_plan_request_id):
             return self.error('Observation plan not yet available.')
 
 
+class ObservationPlanFieldsHandler(BaseHandler):
+    @auth_or_token
+    def delete(self, observation_plan_request_id):
+        """
+        ---
+        description: Get GeoJSON summary of the observation plan.
```
Comment (jadalilleboe) [inline skyportal/handlers/api/observation_plan.py:912]: Is this the right description?

**Review by jadalilleboe — APPROVED** [review]
LGTM, one documentation comment

File: skyportal/handlers/api/observation_plan.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/handlers/api/observation_plan.py]: oops!

File: skyportal/handlers/api/observation_plan.py [hunk 2]
```diff
@@ -904,6 +904,81 @@ def get(self, observation_plan_request_id):
...
+              List of field IDs to remove from the plan
+        responses:
+          200:
+            content:
+              application/json:
+                schema: SingleObservationPlanRequest
+        """
+
+        data = self.get_json()
+        field_ids_to_remove = data.pop('fieldIds', None)
```
Comment (guynir42) [inline skyportal/handlers/api/observation_plan.py:936]: This should probably be a `get_query_argument` as the docstring suggests this is a query argument (which sounds right).

File: static/js/components/ObservationPlanRequestLists.jsx [hunk 3]
```diff
@@ -131,6 +137,12 @@ const ObservationPlanGlobe = ({ observationplanRequest, loc }) => {
             height={300}
             width={300}
           />
+          <Button
+            variant="contained"
+            onClick={() => handleDeleteObservationPlanFields(obsList)}
+          >
+            Delete selected from observation plan
```
Comment (guynir42) [inline static/js/components/ObservationPlanRequestLists.jsx:144]:
```suggestion
            Delete selected fields from observation plan
```

**Review by guynir42 — COMMENTED** [review]
Some small corrections:

**Pull request merged**